papers/eutxo/formal-model.tex

```diff
@@ -49,20 +49,22 @@ \subsection{\EUTXO{}: Enhanced scripting}
 The \EUTXO{} model adds a number of new features to the model
 proposed in Zahnentferner~\cite{Zahnentferner18-UTxO}.
 
-The first change is that we allow transaction outputs to carry a \emph{data value}
-along with the validator, which is passed in as an additional argument during
-validation.
-This allows a contract to carry some state (the data) without changing
-its ``code'' (the validator). We will use this to carry the state of our
-state machines (see Section~\ref{sec:informal-eutxo}).
-
-The second change is that the validator receives some information about the
-transaction that is being validated. This information is passed in as an
-additional argument of type \ptx{}. The additional information enables the
-validator to enforce much stronger conditions than is possible with a bare
-\UTXO{} model --- in particular, it can inspect the \emph{outputs}
-of the current transaction, which is essential for ensuring contract
-continuity (see Section~\ref{sec:informal-eutxo}).
+The first change is that as well as the validator we allow transaction
+outputs to carry a piece of data called the \emph{datum}, which is
+passed in as an additional argument during validation.  This allows a
+contract to carry some state (the datum) without changing its ``code''
+(the validator). We will use this to carry the state of our state
+machines (see Section~\ref{sec:informal-eutxo}).
+
+The second change is that the validator receives some information
+about the transaction that is being validated. This information, which
+we call the \textit{validation context}, is passed in as an additional
```

Comment:
I'd say just the "context" here, if we're really going to call it that. If you want to disambiguate, perhaps say `the \emph{context} of the validation`?

Reply:
Yeah, I took that out.  I'll mention it in the main EUTXO document though.